config/manifests/bases/open-liberty.clusterserviceversion.yaml

```diff
@@ -16,6 +16,7 @@ metadata:
     features.operators.openshift.io/token-auth-azure: "false"
     features.operators.openshift.io/token-auth-gcp: "false"
     olm.skipRange: '>=0.8.0 <1.4.0'
+    operators.openshift.io/infrastructure-features: '["disconnected"]'
```

Comment:
Red Hat's pipeline doesn't like this deprecated annotation. So we can only keep it for WLO.